src/renderer/renderer.cpp

```diff
@@ -61,6 +60,7 @@ const auto SHADER_PREAMBLE = R"shd(
 const auto SHADER_PREAMBLE = R"shd(
 #version 150
 
+precision highp float;
```

Comment:
This is the shader preamble for regular (non-ES) OpenGL, so it shouldn't need any changes, as it won't be used if `RIGEL_USE_GL_ES` is defined